src/site/antora/modules/ROOT/pages/manual/thread-context.adoc

```diff
@@ -120,9 +120,25 @@ executor.submit(() -> {
 [#config]
 == Configuration
 
-You can configure thread context using following properties:
+Since the thread context is inherently linked to the logging implementation, its configuration options depend on the logging implementation used:
 
-include::partial$manual/systemproperties/properties-thread-context.adoc[leveloffset=+1]
+Simple Logger::
++
+See xref:manual/simple-logger.adoc#thread-context[Thread context configuration of Simple Logger].
+
+Log4j Core::
++
+See xref:manual/systemproperties.adoc#properties-thread-context[Thread context configuration of Log4j Core].
+
+SLF4J implementations (Logback)::
```

Comment:
I find this a little bit confusing, since `log4j-slf4j2-impl` is also an SLF4J implementation. Can we (shall we?) clarify this a little bit more in the docs?

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2690/commits/33ba51e2dc74d2901f529f79d0deeffa69b47235